.github/workflows/publish-android.yml

```diff
@@ -70,7 +70,17 @@ jobs:
             echo "PUB_MODE=-PSNAPSHOT" >> $GITHUB_ENV
           fi
 
-      - name: Gradle Publish Android Package
-        run: ./gradlew publishAndroidReleasePublicationToGithubPackagesRepository -Pandroid=true ${{ env.PUB_MODE }}
+      - name: Gradle Publish Android Package to GitHub packages repository
+        run: ./gradlew publishAndroidReleasePublicationToGithubPackagesRepository -PremotePublication=true -Pandroid=true ${{ env.PUB_MODE }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PGP_SIGNING_KEY: ${{ secrets.PGP_SIGNING_KEY }}
+          PGP_SIGNING_PASSWORD: ${{ secrets.PGP_SIGNING_PASSWORD }}
```

Comment:
Looks like we can use the `secrets.SSH_PRIVATE_KEY` and `secrets.SSH_PASSPHRASE` here instead. Could you try that? I filed a request to have the GPG keys added but they told me to use the same one already in the org secrets.